api/src/main/java/org/eclipse/microprofile/config/Config.java

```diff
@@ -69,13 +69,11 @@
      *
      * @param propertyName
      *             The configuration propertyName.
-     * @return The resolved property value as a String-Optional, which will bypass converters.
+     * @return The resolved property value as a String, which will bypass any converters.
      *
-     * @throws IllegalArgumentException
-     *             is thrown if the propertyName maps to an object that is not a
-     *             String.
+     * 
      */
-    Optional<String> getString(String propertyName);
+    String getRawString(String propertyName);
```

Comment:
> Since converters work with String, it's natural to return raw value as a String.

That's what I meant above, I doubt basing it on String is the right thing to do. I've opened #87 to discuss this separately.